nearuplib/localnet.py

```diff
@@ -57,32 +57,25 @@ def run(binary_path, home, num_nodes, num_shards, override, verbose=True, intera
                    validators=num_nodes,
```

Comment:
`run_binary()` needs a string for the `home` argument, so maybe change it to `str(home)`? (I can't figure out how to attach this comment to a hunk thats not part of the PR...)

Reply:
No it doesn’t? Note that subprocess.Popen accepts a list of Union[str, PathLike].

Reply:
The string requirement comes from this part:

```
    if print_command:
        print(f'Running "{" ".join(command)}"')
```

Reply:
But yea maybe makes more sense to just change that function than to change the arg to `str(home)`